src/MultiFormats.Tests/MultiHashTest.cs

```diff
@@ -0,0 +1,558 @@
+                Algorithm = "keccak-224",
+                Input = "beep boop",
+                Output = "1a1c2bd72cde2f75e523512999eb7639f17b699efe29bec342f5a0270896"
+            },
+            new TestVector
+            {
+                Algorithm = "keccak-256",
+                Input = "beep boop",
+                Output = "1b20ee6f6b4ce5d754c01203cb3b99294b88615df5999e20d6fe509204fa254a0f97"
+            },
+            new TestVector
+            {
+                Algorithm = "keccak-384",
+                Input = "beep boop",
+                Output =
+                    "1c300e2fcca40e861fc425a2503a65f4a4befab7be7f193e57654ca3713e85262b035e54d5ade93f9632b810ab88b04f7d84"
+            },
+            new TestVector
+            {
+                Algorithm = "keccak-512",
+                Input = "beep boop",
+                Output =
+                    "1d40e161c54798f78eba3404ac5e7e12d27555b7b810e7fd0db3f25ffa0c785c438331b0fbb6156215f69edf403c642e5280f4521da9bd767296ec81f05100852e78"
+            },
+            new TestVector
+            {
+                Algorithm = "blake2b-512",
+                Input = "beep boop",
+                Output =
+                    "c0e402400eac6255ba822373a0948122b8d295008419a8ab27842ee0d70eca39855621463c03ec75ac3610aacfdff89fa989d8d61fc00450148f289eb5b12ad1a954f659"
+            },
+            new TestVector
+            {
+                Algorithm = "blake2b-160",
+                Input = "beep boop",
+                Output = "94e40214fe303247293e54e0a7ea48f9408ca68b36b08442"
+            },
+            new TestVector
+            {
+                Algorithm = "blake2s-256",
+                Input = "beep boop",
+                Output = "e0e402204542eaca484e4311def8af74b546edd7fceb49eeb3cdcfd8a4a72ed0dc81d4c0"
+            },
+            new TestVector
+            {
+                Algorithm = "dbl-sha2-256",
+                Input = "beep boop",
+                Output = "56209cd9115d76945c2455b1450295b05f4edeba2e7286bc24c23e266b48faf578c0"
+            },
+            new TestVector
+            {
+                Algorithm = "identity",
+                Input = "ab",
+                Output = "00026162"
+            },
+            new TestVector
+            {
+                Algorithm = "id",
+                Input = "ab",
+                Output = "00026162"
+            },
+        };
+
+        [TestMethod]
+        public void CheckMultiHash()
+        {
+            foreach (var v in TestVectors)
+            {
+                if (v.Ignore) continue;
+                var bytes = Encoding.UTF8.GetBytes(v.Input);
+                var mh = MultiHash.ComputeHash(bytes, v.Algorithm);
+                Assert.AreEqual(v.Output, mh.ToArray().ToHexString(), v.Algorithm);
+            }
+        }
+
+        [TestMethod]
+        public void CheckMultiHash_Stream()
+        {
+            foreach (var v in TestVectors)
+            {
+                if (v.Ignore) continue;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)